features/docker/config.pan

```diff
@@ -12,15 +12,15 @@ required = No
 
 include 'features/docker/core';
 
-include if ( DOCKER_BACKUP ) 'features/docker/backup';
+include if (DOCKER_BACKUP) 'features/docker/backup';
 
-include if( DOCKER_PIPEWORK ) 'features/docker/pipework';
+include if (DOCKER_PIPEWORK) 'features/docker/pipework';
 
 # Configure Docker YUM repository if necessary
 variable SITE_REPOSITORY_LIST = {
-    if ( is_defined(DOCKER_YUM_REPOSITORY) ) {
-        SELF[length(SELF)] = format('%s', DOCKER_YUM_REPOSITORY);
+    repolist = SELF;
+    if (is_defined(DOCKER_YUM_REPOSITORY)) {
+        repolist = append(repolist, to_string(DOCKER_YUM_REPOSITORY));
```

Comment:
or just `append`, no temp variables or anything (but i know you don't like that way of using append)